### ISSUE TITLE
Abour readme

### ISSUE THREAD

**Issue by yunzi-94** [JiapengWu/TeMP#1]
Thanks for sharing the codes. But actually, it is hard to follow and understand the codes without a useful readme file.

**Comment by JiapengWu** [comment]
Hi Yunzi,


I am deeply sorry about the delay for documentation. I have been through a rough period and I finally got some energy to sit down and update the repository. I hope this is still useful.

Best,

Paul

On Dec 20, 2020, at 11:08 PM, yunzi-94 ***@***.******@***.***>> wrote:



Thanks for sharing the codes. But actually, it is hard to follow and understand the codes without a useful readme file.
